egs/zeroth_korean/s5/local/updateSegmentation.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2017 Lucas Jo (Atlas Guide)
+# Apache 2.0
+
+# do this when the segmentation rule is changed
+dataDir=$1
+lmDir=$2
+
+exists(){
+	command -v "$1" >/dev/null 2>&1
+}
+
+# check morfessor installation 
+if ! exists morfessor; then
+	echo "Please, install Morfessor"
+	exit 1
+fi
+
+trans=$dataDir/text
+echo "Re-segment transcripts: $trans --------------------------------------------"
+if [ ! -f $trans ]; then
+	echo "transcription file is not found in "$dataDir
+	exit 1
+fi
+cp $trans $trans".old"
+awk '{print $1}' $trans".old" > $trans"_tmp_index"
+cut -d' ' -f2- $trans".old" |\
```

Comment:
before using external scrips like morfessor you should add a script in somewhere like tools/extras/ that can automatically install it, and make it so your script can use it from that location.

Reply:
I added a new pull request for morfessor installation script, Dan.
plz check the PR
https://github.com/kaldi-asr/kaldi/pull/2299